src/drivers/sdl/lv_sdl_window.h

```diff
@@ -55,6 +55,8 @@ void * lv_sdl_window_get_renderer(lv_display_t * disp);
 
 void lv_sdl_quit(void);
 
+void * lv_sdl_window_get_window(lv_display_t * disp);
```

Comment:
Don't use an opaque pointer here add an `#include LV_SDL_INCLUDE_PATH` at the top of the file after `#if LV_USE_SDL` and then:

```suggestion
SDL_Window  * lv_sdl_window_get_window(lv_display_t * disp);
```

Reply:
hmm, then the same would apply for lv_sdl_window_get_renderer, no ?